Unbox.swift

```diff
@@ -49,7 +49,7 @@ public func Unbox<T: Unboxable>(data: NSData, context: Any? = nil) -> T? {
 }
 
 /// Unbox binary data into an array of `T`, while optionally using a contextual object
-public func Unbox<T: Unboxable>(data: NSData, context: Any? = nil) -> [T]? {
+public func Unbox<T: Unboxable>(data: NSData, context: Any? = nil, skipInvalidData: Bool = false) -> [T]? {
```

Comment:
I actually think this is the better approach :) See my comment on the PR itself.